### PR TITLE
Implemented the support for proxy configuration from player

### PR DESCRIPTION
**Pull request by rodrigopavezi**
Implemented the reading of the RiseDisplayNetworkII.ini properties file
Passed the proxy configuration to the request module calls
Implemented handling of errors to connect through the proxy
Implemented handling of the errors to get the file or metadata from the host server. It will return 502 if not possible to connect, 504 if it cannot reach the server and 534 if it gets a 404 from the server.

Added tests and updated existent ones

@stulees please review. cheers

### CODE REVIEW

File: app/routes/file.js [hunk 1]
```diff
@@ -109,15 +116,28 @@ const FileRoute = function(app, headerDB, updateDuration) {
     console.info("File exists in cache. Not downloading", fileUrl, new Date());
   }
 
-  function sendResponse(res, fileUrl) {
-    res.status(202)
-      .send({
-        status: 202,
-        message: "File is downloading"
-      });
+  function sendInvalidResponseResponse(res, fileUrl, statusCode) {
```
Comment (stulees) [inline app/routes/file.js]: @rodrigopavezi I think there's an extra "Response" in this function name.

Comment (rodrigopavezi) [inline app/routes/file.js]: @stulees I meant that. It means this function will send a response to the client regarding an invalid response from the host server. cheers

Comment (stulees) [inline app/routes/file.js]: @rodrigopavezi Oh okay, cheers

**Comment by stulees** [comment]
@rodrigopavezi Looks good 👍

**Pull request merged**